lib/module/xkcd.go

```diff
@@ -0,0 +1,70 @@
+package module
+
+import (
+	"fmt"
+	"github.com/davidscholberg/irkbot/lib/configure"
+	"github.com/davidscholberg/irkbot/lib/message"
+	"github.com/nishanths/go-xkcd"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const apiUrlFmtDefine = "https://relevantxkcd.appspot.com/process?%s"
+
+func Helpxkcd() []string {
+	s := "xkcd <search> - find a xkcd comic relevant to the search term"
+	return []string{s}
+}
+func get(apiURL string) (string, error) {
+	response, err := http.Get(apiURL)
+	if err != nil {
+		return "", err
+	}
+	defer response.Body.Close()
+	bodyBytes, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return "", err
+	}
+	bodyString := string(bodyBytes)
+	return bodyString, err
+}
+func fmtString(bodyString string) string {
+	bodyStrings := strings.Split(bodyString, "\n")
+	spacedStrings := strings.Fields(bodyStrings[2])
```

Comment:
Don't assume that `bodyStrings` has at least 3 elements. Check first, and if it has less than 3, then return an err object to the calling function. Also remember to error check the returned err object in the calling function.